apis/python/src/tiledbsoma/types.py

```diff
@@ -30,4 +30,4 @@
 DenseNdCoordinates = Tuple[DenseCoordinates, ...]
 
 SparseCoordinates = Union[int, slice, Tuple[int, ...], List[int], pa.IntegerArray]
-SparseNdCoordinates = Tuple[DenseCoordinates, ...]
+SparseNdCoordinates = Union[Tuple[DenseCoordinates, ...], List[DenseCoordinates]]
```

Comment:
why not use Sequence, ie,   Sequence[DenseCoordinates] rather than the union of tuple & list?